test/e2e/common/empty_dir.go

```diff
@@ -153,6 +161,37 @@ func doTestSetgidFSGroup(f *framework.Framework, image string, medium v1.Storage
 	f.TestContainerOutput(msg, pod, 0, out)
 }
 
+func doTestSubPathFSGroup(f *framework.Framework, image string, medium v1.StorageMedium) {
+	var (
+		volumePath = "/test-volume"
```

Comment:
consider making this a const usable by various tests. This string is assigned to `volumePath` 8 times.